language/Spanish/strings.po

```diff
@@ -1538,22 +1552,22 @@ msgstr "Posición vertical %1$s"
 #: src/frontend/mame/ui/ui.cpp:1467
 #, c-format
 msgid "Laserdisc '%1$s' Horiz Stretch"
-msgstr "Estiramiento horizontal de Laserdisc «%1$s»"
+msgstr "Estiramiento horizontal de laserdisc «%1$s»"
```

Comment:
I was not sure about "Laserdisc" as a trademark or a technology... Reverted! Thanks!